cmd/snap-seccomp/main.go

```diff
@@ -135,6 +136,11 @@ package main
 //		return htobe64(val);
 //}
 //
+//static int mygetgrnam_r(const char *name, struct group *grp,char *buf,
+//			  size_t buflen, struct group **result) {
+//	return getgrnam_r(name, grp, buf, buflen, result);
```

Comment:
IMO, no because everything we will be using it for will be in the local databases. Systems with network NSS modules always include the local database module first so that the system can boot up or function when the system is down. Consider debugging a broken system with no networking-- you need to be able to sudo/su. Not to mention it would be a terribly non-performant system if it had to reach out over the network for 'root' or 'adm'.

Keeping that in mind, here are the only places we are going to use this:
 1. 'root'
 2. 'daemon'
 3. 'shadow'
 4. snapd managed users

For 1 and 2, both are defined in the LSB as being required on any Linux system (http://refspecs.linuxfoundation.org/LSB_5.0.0/LSB-Core-generic/LSB-Core-generic/usernames.html)

For 3, the shadow database is in place everywhere in practice. If snapd is ported to a system that uses a different name, it is easy to adjust account-control to do `if isNonStandardDistro then use nonstandard-shadow-group`. We can cross that bridge when we get there.

For 4, [snapd will be adding users to the local system via local system commands](https://forum.snapcraft.io/t/multiple-users-and-groups-in-snaps/1461/3) (eg, useradd and groupadd)

Reply:
"Keeping that in mind, here are the only places we are going to use this"

Note, it is technically possible we might introduce an interface to allow modifying something in /etc, like we do for shadow, but that would be exactly the same as for shadow.